### ISSUE TITLE
Warn user about gas limit for the whole block

### ISSUE THREAD

**Issue by 0xGabi** [1Hive/transactions-app#13]
Warn the user that there is a cap on the amount of parallel transactions that could be included in a single action.

**Comment by pakokrew** [comment]
Good catch, do you have an idea of how we can know how much is that limit ?
